_docs/releasing_gem.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Releasing a new version of a Ruby Gem
+date: 2022-09-27
+summary: How to perform a complete version release
+---
+
+The release process is split into two parts. Part one can be done by anybody
+with a GitHub account. You do not need to be part of the Vox Pupuli GitHub
+organisation.
+
+Part One: Create a 'release pr'
+This pull request updates the changelog and bumps the version number to the
+target version. The version is set in the gemspec file in the root of the git
+repository. Sometimes it's a variable coming from something like
+`lib/$gem/version.rb`. An example is [beaker](https://github.com/voxpupuli/beaker/blob/f60ac9413f9c7976a6645ef9e1dd2afbcc6542de/beaker.gemspec#L8),
+([version.rb](https://github.com/voxpupuli/beaker/blob/master/lib/beaker/version.rb#L3)).
+
+Now you can install the changelog generator:
+
+```bash
+bundle install --jobs $(nproc) --path .vendor/ --with release
```

Comment:
I think passing `--path` and `--with` to `bundle install` is deprecated. Also, for parallelism I'd suggest referring to some common documentation. `export BUNDLE_JOBS="$(nproc)"` in my `~/.bashrc`.

Reply:
yes it's deprecated. I need to check for the new syntax, I cannot keep that in my head :D

Reply:
I updated this